src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -142,6 +144,20 @@ public VerboseProcess(final Process prc, final Level stdout,
         if (stderr == null) {
             throw new IllegalArgumentException("stderr LEVEL can't be NULL");
         }
+        final String format = new StringBuilder("%s LEVEL can't be set to ALL ")
```

Comment:
@prondzyn did you read http://www.yegor256.com/2014/06/19/avoid-string-concatenation.html#apache-stringutils-join? There is suggestion to use `StringUtils.join` for long strings